src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![deny(warnings)]
 #![recursion_limit = "128"]
 
+extern crate byteorder;
```

Comment:
Same question as above.

Reply:
Yeah, the `extern crate` statements should be in alphabetical order.